apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -580,6 +633,18 @@ const Ticket = new GQLListSchema('Ticket', {
             // Todo(zuch): add placeClassifier, categoryClassifier and classifierRule
             if (!hasDbFields(['organization', 'source', 'status', 'details'], resolvedData, existingItem, context, addValidationError)) return
 
+            const user = get(context, ['req', 'user'])
+            const userType = get(user, 'type')
+
+            if (userType === RESIDENT && operation === 'create') {
+                await checkDailyTicketLimit({
+                    phone: get(user, 'phone'),
```

Comment:
It's better to use a `user.id` here. Because the phone is not required field.

Reply:
Ok, nice catch